etcd_helper/etcdfuncs.go

```diff
@@ -0,0 +1,113 @@
+package etcd_helper
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/coreos/etcd/client"
+)
+
+type Etcdreader struct {
+	etcdServiceURL string
+}
+type Etcdwriter struct {
+	etcdServiceURL string
+}
+
+func (o Etcdreader) get(resourceKey string) (string, error) {
+	cfg := client.Config{
+		Endpoints: []string{o.etcdServiceURL},
+		Transport: client.DefaultTransport,
+	}
+	c, err := client.New(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	kapi := client.NewKeysAPI(c)
+
+	resp, err1 := kapi.Get(context.Background(), resourceKey, nil)
+	if err1 != nil {
+		return "", fmt.Errorf("Error: %v", err1)
+	} else {
```

Comment:
We don't need else block. We can put the debug logging statement with some debug log level so that we can control whether the log line gets printed or not.